### PR TITLE
ANSIENG-4174 CP Ansible Support for Ubuntu 22.04

### PR DESCRIPTION
**Pull request by mansi-jain-1206**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?: 
[Kraft + AMD](https://semaphore.ci.confluent.io/workflows/9e201207-dd1f-4989-b41a-073e20920d3f?pipeline_id=95adacd3-90b2-495a-b241-8d20c4ce8311)

[ZK + ARM](https://semaphore.ci.confluent.io/workflows/d8ec8036-3bb5-478b-a13a-bb90a58d879a?pipeline_id=c3e59d41-9de9-418c-96f6-e53a81595cf5
)

# Checklist:

- [ ] Any variable/code changes have been validated to be backwards compatible (doesn't break upgrade)
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] If required, I have ensured the changes can be discovered by cp-ansible discovery codebase
- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by rrbadiani** [comment]
Can you please add the link of semaphore test runs for these scenarios

File: molecule/Dockerfile-ubuntu-java11.j2 [hunk 1]
```diff
@@ -2,7 +2,8 @@ FROM {{ item.image }}
 
 RUN mkdir -p /usr/share/man/man1 && \
     apt-get update -y && \
-    apt-get install -y openjdk-11-jdk wget gnupg
+    apt-get install -y openjdk-11-jdk wget gnupg  && \
+    apt-get install -y dirmngr
```
Comment (rrbadiani) [inline molecule/Dockerfile-ubuntu-java11.j2:6]: whats the reason for adding dirmgr package ?

File: molecule/archive-plain-debian10/molecule.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@ platforms:
       - ${CONTROLLER_HOSTGROUP:-zookeeper}
       - ${CONTROLLER_HOSTGROUP:-zookeeper}_migration
     image: geerlingguy/docker-debian10-ansible
-    dockerfile: ../Dockerfile-debian10-archive.j2
+    dockerfile: ../Dockerfile-debian10.j2
```
Comment (rrbadiani) [inline molecule/archive-plain-debian10/molecule.yml:15]: Whats the reason for modifying this dockerfile ?

File: molecule/Dockerfile-ubuntu-java11.j2 (hunk 1 above)
Comment (mansi-jain-1206) [reply molecule/Dockerfile-ubuntu-java11.j2:6]: Docker file Dockerfile-ubuntu-java11.j2 and Dockerfile-ubuntu2004-java11.j2 were exactly same apart from this dirmgr package, so added this in one docker file and deleted other to create no impact.

File: molecule/archive-plain-debian10/molecule.yml (hunk 2 above)
Comment (mansi-jain-1206) [reply molecule/archive-plain-debian10/molecule.yml:15]: There were duplicate docker files, so cleaned them up.

**Review by rrbadiani — APPROVED** [review]
LGTM, Thanks!

**Comment by mansi-jain-1206** [comment]
Closing this PR and opening again.

**Pull request closed without merging**